corgi/tasks/pyxis.py

```diff
@@ -58,7 +96,6 @@ def _slow_fetch_pyxis_manifest_for_repository(
 ) -> bool:
 
     # According to the PURL spec, the name part is the last fragment of the repository url
```

Comment:
Ah, I think I forgot to make a small doc update. When I saw "fragment" written here, I thought it was referring to URL fragments like `/path/to/page#fragment_link_to_some_heading. Maybe below is more clear:
```suggestion
    # According to the PURL spec, the name part is the last slash-separated path part of the repository url
```

Reply:
fixed